src/projects/EnsureThat/Enforcers/EnumArg.cs

```diff
@@ -37,5 +39,25 @@ public sealed class EnumArg
 
             return value;
         }
+
+        /// <summary>
+        /// Confirms that the <paramref name="value"/> is defined in the enum <typeparamref name="T"/>.
+        /// Supports <see cref="FlagsAttribute"/> attribute.
+        /// </summary>
+        public T IsDefinedExtended<T>(T value, [InvokerParameterName] string paramName = null, OptsFn optsFn = null) where T : struct, Enum
```

Comment:
[nit] sorry to be so pedantic. I think it'd be good to include Flags somewhere in the name of this method. Spitballing here but maybe something like

AreFlagsDefined() (argument: since we are checking >1 value)
IsFlagsDefined() (argument: since the 'flags' is a single thing on the enum)
IsDefinedWithFlagsSupport() (argument: since we have IsDefined and the difference here is "with flags support" + discover-ability of the same prefix)

Reply:
I just struggle to "grok" what "extended" means without having to open up the xmldoc, which diff tools don't show you the documentation at the usage site

Reply:
Was planning on merging this one now, but noticed this comment.

"Extended" as in: "_We do some magical stuff here, e.g. somewhat supporting flags, and you should read the docs._"

But perhaps `IsDefinedWithFlagsSupport()` is better. @nevaldas: what do you think?

Reply:
Yeah, this one is better. Renamed. ;)